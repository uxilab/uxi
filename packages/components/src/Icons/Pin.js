import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Pin = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 9.29 16"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M4.65,0A4.65,4.65,0,0,0,0,4.65c0,2.19,3.25,9.44,3.9,10.87a.82.82,0,0,0,1.49,0c.4-.88,3.9-8.63,3.9-10.87A4.65,4.65,0,0,0,4.65,0Zm.13,12.87-.13.3-.13-.3C3.84,11.3,1.63,6.06,1.63,4.65a3,3,0,0,1,6,0C7.66,6.07,5.45,11.3,4.78,12.87Z" />
          <circle cx="4.65" cy="4.65" r="1.29" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Pin.propTypes = {
  color: PropTypes.string,
};

Pin.defaultProps = {
  color: '#6d6d71',
};

export default Pin;
