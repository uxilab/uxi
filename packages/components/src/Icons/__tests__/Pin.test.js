import React from 'react';
import Pin from '../Pin';

describe('Icon : <Pin />', () => {
  it('match snapshot', () => {
    expect(shallow(<Pin />)).toMatchSnapshot();
  });
});

