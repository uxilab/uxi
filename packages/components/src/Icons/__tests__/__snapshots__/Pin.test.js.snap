// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Pin /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 9.29 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M4.65,0A4.65,4.65,0,0,0,0,4.65c0,2.19,3.25,9.44,3.9,10.87a.82.82,0,0,0,1.49,0c.4-.88,3.9-8.63,3.9-10.87A4.65,4.65,0,0,0,4.65,0Zm.13,12.87-.13.3-.13-.3C3.84,11.3,1.63,6.06,1.63,4.65a3,3,0,0,1,6,0C7.66,6.07,5.45,11.3,4.78,12.87Z"
        />
        <circle
          cx="4.65"
          cy="4.65"
          r="1.29"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
